internal/config/config.go

```diff
@@ -146,25 +146,17 @@ type Config struct {
 	// LIBRARIAN_GITHUB_TOKEN environment variable.
 	GitHubToken string
 
-	// GitUserEmail is the email address used in Git commits. It is used in
-	// all commands that create commits in a language repository:
-	// create-release-pr, configure, update-apis and update-image-tag.
-	//
-	// GitUserEmail is optional, with a default value of noreply-cloudsdk@google.com
-	// being used for commits if it's unspecified.
+	// PushConfig specifies the email address and display name used in Git commits,
+	// in the format "name:email".
 	//
-	// GitUserEmail is specified with the -git-user-email flag.
-	GitUserEmail string
-
-	// GitUserName is the display name used in Git commits. It is used in
-	// all commands that create commits in a language repository:
+	// PushConfig is used in all commands that create commits in a language repository:
 	// create-release-pr, configure, update-apis and update-image-tag.
 	//
-	// GitUserName is optional, with a default value of "Google Cloud SDK"
-	// being used for commits if it's unspecified.
+	// PushConfig is optional. If unspecified, commits will use a default name of
+	// "Google Cloud SDK" and a default email of noreply-cloudsdk@google.com.
```

Comment:
According to @codyoss 's reply to [this comment](https://docs.google.com/document/d/1bQ5aXIfQEauI6cHlZKh8R6PBAG_sfHRaDxHcxRCQMYI/edit?resourcekey=0-TyGOVF1t5hGtF0Pydz572w&disco=AAABk4ECl78), if unspecified, we do not want to create commit

Reply:
If you think it should be in a separate PR, can you create an issue for it and link it to a TODO?